README.md

```diff
@@ -8,8 +8,10 @@ Shippable CI image for Java on Ubuntu 14.04. Available jdk versions:
 3. openjdk10
 4. openjdk11
 5. oraclejdk8
-6. oraclejdk11
+6. oraclejdk11.0.1
```

Comment:
I think we should not change this. The user still needs to specify `oraclejdk11` in their YML, so this will start getting confusing.